cmd/kcp-config/main.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"fmt"
+	"gopkg.in/yaml.v3"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"syscall"
+)
+
+func main() {
+	if len(os.Args) < 3 {
+		fmt.Printf("usage: kcp-config [config-file] [cmd] [args...]\n")
+		os.Exit(1)
+	}
+
+	fname := os.Args[1]
+
+	data, err := ioutil.ReadFile(fname)
+	if err != nil {
+		fmt.Printf("Error reading config file %s: %v\n", fname, err)
+		os.Exit(1)
+	}
+
+	contents := map[string]interface{}{}
+	err = yaml.Unmarshal([]byte(data), &contents)
+	if err != nil {
+		fmt.Printf("error: %v\n", err)
+		os.Exit(1)
+	}
+
+	args := os.Args[2:]
+	for k, v := range contents {
+		args = append(args, fmt.Sprintf("--%s=%v", k, v))
+	}
+
+	cmd, err := exec.LookPath(os.Args[2])
```

Comment:
When can this be anything other than `kcp`? Why is it user-configurable?

Reply:
I'm thinking this wrapper could be used for any of the commands in this project that we'd want to run in a container (and have their args supplied in a `ConfigMap`).